src/dispatch/static/dispatch/src/dashboard/case/CaseTypeBarChartCard.vue

```diff
@@ -40,17 +40,15 @@ export default {
   data() {
     return {
       types: [],
+      counts: [],
     }
   },
 
   created: function () {
-    CaseTypeApi.getAll({ itemsPerPage: -1 }).then((response) => {
-      this.types = map(
-        filter(response.data.items, function (item) {
-          return !item.exclude_from_metrics
-        }),
-        "name"
-      )
+    CaseApi.getTypeCount().then((response) => {
```

Comment:
```suggestion
    CaseTypeApi.getTypeCount().then((response) => {
```

would need to move method to apply